settings.gradle.kts

```diff
@@ -74,3 +74,5 @@ include("integration-test-common")
 include(":bundles:aws-bundle")
 include(":bundles:gcp-bundle")
 include(":bundles:aliyun-bundle")
+include("bundles:azure-bundle")
+findProject(":bundles:azure-bundle")?.name = "azure-bundle"
```

Comment:
Why do we need this line?

Reply:
It's automatically added by IDE and I will remove it.